packager/live_packager_test.cc

```diff
@@ -168,9 +168,9 @@ TEST_F(LivePackagerBaseTest, EncryptionFailure) {
     std::vector<uint8_t> segment_buffer = ReadTestDataFile(segment_num);
     ASSERT_FALSE(segment_buffer.empty());
 
-    FullSegmentBuffer in;
-    in.SetInitSegment(init_segment_buffer.data(), init_segment_buffer.size());
-    in.AppendData(segment_buffer.data(), segment_buffer.size());
+    SegmentData init_seg(init_segment_buffer.data(),
```

Comment:
**nit**: finally need to increase line width for clang-format,  it's annoying to see all these wraps (((

Reply:
Initially I was just following the Contributing Guid for clang-format: 
https://github.com/Pluto-tv/shaka-packager-live/blob/pluto-cmake/docs/source/build_instructions.md#contributing. 

Do we want to change or modify this? Maybe `ColumnLimit: 100`?

Reply:
This is certainly debatable, but I'd respect the standard of the existing codebase and keep this as is.  Just my 2 cents.  We can take a vote on this too.

Reply:
Existing settings make me think about what type of computer shaka developers are using. Right now on a laptop, I have ~70% of free horizontal space with their formatting.

Reply:
This is outside the scope of this PR.  Let's have a follow-up discussion and address this in a separate PR.